Remove instances of LEDGER_INDEX_INFO

emord opened the pull request:
https://github.com/dimagi/commcare-hq/pull/25508#issuecomment-549544587

emord committed:
Remove instances of LEDGER_INDEX_INFO

emord committed:
Remove last LEDGER_INDEX_INFO reference

emord committed:
Remove ledgers reference in prod_es_indices

esoergel committed:
Remove ledgers from ptop_preindex

The corresponding reindexers were removed here:
https://github.com/dimagi/commcare-hq/pull/25508/files#diff-e0b21abb59f9252bfd583c575e668500L54

esoergel commented:
Test failure looks real:
```
====================================================================
FAIL: testapps.test_pillowtop.tests.test_settings:PillowtopSettingsTest.test_instantiate_all
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_settings.py", line 30, in test_instantiate_all
    self.assertEqual(expected_meta[pillow_config.name], _pillow_meta_from_config(pillow_config))
AssertionError: {'advertised_name': 'LedgerToElasticsearchP[219 chars]low'} != {'name': 'LedgerToElasticsearchPillow', 'ad[214 chars]eed'}
  {'advertised_name': 'LedgerToElasticsearchPillow',
   'change_feed_type': 'KafkaChangeFeed',
-  'checkpoint_id': 'LedgerToElasticsearchPillow-test_ledgers_2016-03-15',
?                                                -----
+  'checkpoint_id': 'LedgerToElasticsearchPillow-ledgers_2016-03-15',
   'full_class_name': 'pillowtop.pillow.interface.ConstructedPillow',
   'name': 'LedgerToElasticsearchPillow'}
----------------------------------------------------------------------
Ran 3606 tests in 2415.372s
```

emord committed:
Remove test string in pillow checkpoint id

This no longer needs the string test because it does not write to a test
ES index.

emord committed:
Merge pull request #25813 from dimagi/es/ledger-es

Remove ledgers from pillow map

emord committed:
Merge branch 'master' into je/ledger-cleanup
